### PR TITLE
Uplift fix missing unwrap bkey

### PR DESCRIPTION
**Pull request by dschom**
## Because

- We had reports that some users were getting logged out of sync after signing in.
- We had FF logs that indicated this occurred when `keyFetchToken` was provided but `unwrapBKey` was not.
- We found a couple Sentry reports indicating the aforementioned state, and they all were coming through SignInUnblock.

## This pull request

- Makes sure the response from `beginSignIn` is augmented with the correct `unwrapBKey`
- Handles possibility that the request to `getCredentialStatus` fails. 

## Issue that this pull request solves

Closes: FXA-10641

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

This is a quick patch for a train-295 release. There will be another patch to follow that'll add some test coverage and deal with any potential merge conflicts on main.

### CODE REVIEW

File: packages/fxa-settings/src/pages/Signin/SigninUnblock/container.tsx
Comment (LZoog) [inline packages/fxa-settings/src/pages/Signin/SigninUnblock/container.tsx:94]: Just curious, why make this an IIFE instead of `let status;` with `try`/`catch`?

**Review by LZoog — APPROVED** [review]
I didn't manually test this (I might need to refresh myself on how to get to signin unblock state locally, do we have an env flag?) but the code LGTM based on the problem described. I figured we were missing this somewhere and the user had to go through signin unblock so, 👍

File: packages/fxa-settings/src/pages/Signin/SigninUnblock/container.tsx
Comment (dschom) [reply packages/fxa-settings/src/pages/Signin/SigninUnblock/container.tsx:94]: Cleaner IMO

**Pull request merged**
